src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+7.4
+---
+
+* Deprecate evaluating options in the base `Constraint`. Initialize properties in the constructor of the concrete constraint
+  class instead and add the `#[InitializesOptions]` attribute to opt-out of the deprecated behaviour.
```

Comment:
isn't `InitializesOptions` gone now ?

Reply:
indeed, changelog and upgrade file updated accordingly